src/System.Numerics.Vectors/ref/System.Numerics.Vectors.cs

```diff
@@ -305,6 +305,7 @@ public partial struct Vector<T> : System.IEquatable<System.Numerics.Vector<T>>,
         public Vector(T value) { throw null; }
         public Vector(T[] values) { throw null; }
         public Vector(T[] values, int index) { throw null; }
+        public Vector(System.Span<T> values) { throw null; }
```

Comment:
I think that this API can only be in netcoreapp version of this contract.

Reply:
hmmm then looks like that is the root cause of the failures that I'm seeing in CI...

Reply:
I'm not able to figure out how to fix that part... Will wait for guidance.

Between... OOB = "out-of-band"... something similar as 'patch'? Pardon me but...

Reply:
https://twitter.com/terrajobst/status/915679583873114112 has a good description of the terminology.

Reply:
The discussion at https://github.com/dotnet/corefx/issues/25182#issuecomment-357006245 is related to this as well.